include/hpp/constraints/implicit-constraint-set.hh

```diff
@@ -62,16 +62,13 @@ namespace hpp {
           // Handle comparison types
           const ComparisonTypes_t& comp (constraint->comparisonType ());
           for (std::size_t i = 0; i < comp.size(); ++i) {
-            switch (comp[i]) {
```

Comment:
If there any problem with the `switch` ?

Reply:
Actually not. I made this change when ComparisonType became a class and did not revert it when coming back to an enum.